swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
+
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
```

Comment:
Remember that we look at this when the integration tests were failing. When the `inventory` parameter has a default value of `[]` (even when referred to through a class variable), _every_ `Vendor` instance created without an explicit inventory was sharing the same list for their inventory. Python uses the same _reference_ of any default value for every function call missing the parameter. So when we need to support making a parameter optional which would normally need a mutable value, instead, we set an immutable sentinel value as the default, then check for that and make a new mutable instance for each call.